src/net.h

```diff
@@ -640,8 +640,13 @@ enum
     LOCAL_MAX
 };
 
-bool IsPeerAddrLocalGood(CNode *pnode);
-void AdvertiseLocal(CNode *pnode);
+bool IsPeerAddrLocalGood(CNode& pnode);
+
+/**
+ * Add our "best" local address to the batch of addresses we are planning
```

Comment:
If we hear multiple addresses for ourselves from peers, how do we determine "best"?

Reply:
Expanded this comment a bit to refer to `GetLocal`.
Evaluating the security of `GetLocal` is on my TODO list, but for now we can assume it works I guess...